roles/edpm_bootstrap/meta/argument_specs.yml

```diff
@@ -94,3 +94,10 @@ argument_specs:
         description: |
           Shell command that is executed before any packages are installed by the role.
           Can be used to register systems using any arbitrary registration command(s).
+
+      edpm_bootstrap_command_done_file:
+        type: str
+        default: /.bootstrap_command_done
+        description: |
+          The name of the file that will be used as a marker when the edpm_bootstrap_command
```

Comment:
With the leading slash it's a path. It should have the `path` type and the description should match it.

Reply:
:+1: